Docker Compose - unexpected token at 'Invalid Host header'

Hi!
I've download the repository and launched as it's written for docker:

> # Checkout the project
> $ git clone git://github.com/Codeminer42/cm42-central.git
> $ cd cm42-central
> 
> # copy and edit the configuration
> $ cp .env.sample .env
> $ cp config/database.yml.example config/database.yml
> 
> # Prepare container
> $ docker-compose build
> $ docker-compose run --rm web npm install
> $ docker-compose run --rm web bundle exec rake db:create
> $ docker-compose run --rm web bundle exec rake db:migrate
> $ docker-compose run --rm web bundle exec rake db:seed
> 
> # Up container
> $ docker-compose up
> 

I cannot start the app:

![error](https://user-images.githubusercontent.com/905799/27105223-13bb95b4-508f-11e7-8e07-6f4b274f3bb1.png)


The log from the starting up the app is in the attachment - after:
> $ git clone git://github.com/Codeminer42/cm42-central.git
> $ cd cm42-central
> 
> # copy and edit the configuration
> $ cp .env.sample .env
> $ cp config/database.yml.example config/database.yml

There is the log from terminal (running on Mac OS):
[docker-output.txt](https://github.com/Codeminer42/cm42-central/files/1072937/docker-output.txt)


Thank you in advance.

Hey @maralik thanks for letting us know about this and for the detailed report.
I saw at your log that `webpack` wasn't installed properly, have you tried to install it separately? 

`Error: Cannot find module 'webpack'`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [Unreleased]
+### Fixed
+- Fixing compatibility of docker-composer with webpack
 
 ## [1.5.0] 2017-06-23
 ### Added

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -59,7 +59,7 @@ services:
     volumes:
       - .:/app
       - node_modules:/app/node_modules
-    command: webpack-dev-server --hot --inline --progress --host 0.0.0.0 --config config/webpack.config.js
+    command: webpack-dev-server --hot --inline --progress --host 0.0.0.0 --public webpack --config config/webpack.config.js
 
 volumes:
   db:

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -59,7 +59,7 @@ services:
     volumes:
       - .:/app
       - node_modules:/app/node_modules
-    command: webpack-dev-server --hot --inline --progress --host 0.0.0.0 --public webpack --config config/webpack.config.js
+    command: yarn webpack
 
 volumes:
   db:

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -59,7 +59,7 @@ services:
     volumes:
       - .:/app
       - node_modules:/app/node_modules
-    command: yarn webpack
+    command: yarn run webpack
 
 volumes:
   db:

--- a/package.json
+++ b/package.json
@@ -4,7 +4,8 @@
   "license": "MIT",
   "scripts": {
     "test": "NODE_ENV=test karma start config/karma.conf.js",
-    "coveralls": "cat ./js_coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./js_coverage"
+    "coveralls": "cat ./js_coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./js_coverage",
+    "webpack": "webpack-dev-server --hot --inline --progress --host 0.0.0.0 --public webpack --config config/webpack.config.js"
   },
   "dependencies": {
     "babel-core": "^6.22.1",

